app.env

```diff
@@ -0,0 +1,10 @@
+HOST=localhost
+PORT=5432
+DB_NAME=postgres
+DB_USER=postgres
+DB_PASSWORD=demo
+ADMIN_USERNAME=ali
+ADMIN_PASSWORD=1234
+LOG_LEVEL=2
```

Comment:
It's better to use string representation of log levels. i.e. `debug`, `info`, `error, etc.